### ISSUE TITLE
feature request: select other font for title and timestamp

### ISSUE THREAD

**Issue by pakLebah** [justinfrankel/licecap#45]
Some people might like the current font old-fashion pixelated font. But I'd like to use more modern fonts. At least, something like Arial or Helvetica or San Fransisco (on Mac). It'd be better if we could pick any fonts from standard font dialog of the operating system.

Thank you.

**Comment by tomer-ben-david** [comment]
Yes also the font is very very small especially when inserting new ones!
